### PR TITLE
Added more comprehensive testing/performance questions.

### PR DESCRIPTION
**Pull request by jacopotarantino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing javascript code in a language that compiles to javascript?
```
Comment (darcyclarke) [inline README.md]: Can you make sure to capitalize "JavaScript"?

File: README.md [hunk 2]
```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
...
+
+#### Testing and Performance Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the different between a unit test and a functional/integration test?
+* What tools would you use to find a performance bug in your code(CPU spike, memory leak, etc.)?
+* What are some ways you can improve your website's scroll performance if it is under 60 frames per second?
```
Comment (darcyclarke) [inline README.md]: I think you can be less specific here. No need to mention 60 frames per second. I think simply `What are some ways you may improve your website's scrolling performance?` would be sufficient

File: README.md [hunk 3]
```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
...
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing javascript code in a language that compiles to javascript?
+
+#### Testing and Performance Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the different between a unit test and a functional/integration test?
+* What tools would you use to find a performance bug in your code(CPU spike, memory leak, etc.)?
```
Comment (darcyclarke) [inline README.md]: I wouldn't allude to an answer (ie. the references to CPU spikes, memory leaks etc.).

File: README.md [hunk 4]
```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing javascript code in a language that compiles to javascript?
+
+#### Testing and Performance Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the different between a unit test and a functional/integration test?
```
Comment (darcyclarke) [inline README.md]: I think you meant `difference` not `different`.

Comment (jacopotarantino) [inline README.md]: Bah. I even spell checked the PR haha. Updated request coming in a moment.

**Comment by jacopotarantino** [comment]
Updated to reflect recommended changes. Let me know if you catch anything else.

File: README.md [hunk 5]
```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+
+#### Testing and Performance Questions:
```
Comment (arthurvr) [inline README.md]: >  Testing and Performance

In what way do you see these topics as related here? I feel like they both deserve separated sections, don't you?

Comment (jacopotarantino) [inline README.md]: Hmm... I would probably lump them together under the umbrella of "good code" or something like that but, yeah, you're right. They can be split into 2 sections.

**Comment by jacopotarantino** [comment]
Accidentally closed. @darcyclarke @arthurvr looks good to merge?

File: README.md [hunk 6]
```diff
@@ -136,6 +135,20 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
...
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
+* What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+
+#### Testing Questions:
+
+* What are some advantages/disadvantages to testing your code?
+* What tools would you use to test your code's functionality?
+* What is the difference between a unit test and a functional/integration test?
+* What is the purpose of a code style linting tool?
```
Comment (arthurvr) [inline README.md]: Braindump but maybe we should have some more code quality questions (not for this PR though)

**Comment by arthurvr** [comment]
punting the merge to @h5bp/frontend-interview but it looks better now :D Thanks for working on this!

**Comment by richgilbank** [comment]
LGTM. Thanks for this @jacopotarantino!

**Pull request merged**